NAMESPACE

```diff
@@ -19,4 +19,6 @@ import(formatters)
 import(osprey)
 import(shiny)
 import(teal)
+importFrom(teal.reporter,simple_reporter_srv)
+importFrom(teal.reporter,simple_reporter_ui)
```

Comment:
all `simple_reporter_srv` and `simple_reporter_ui` are prefixed by `teal.reporter::`. Why do we need this import?

Reply:
I was trying to fix this WARNING in R CMD CHECK (for different function back then) https://github.com/insightsengineering/teal.osprey/actions/runs/5598928237/jobs/10239237480#step:37:96 
Will try to update without imports, to triggers this again

Reply:
reverted in 0017981490637da404771895fedf70a540246c0e - let's see the GHA build status

Reply:
oh my, all check have passed